src/IO/importUBC.jl

```diff
@@ -84,6 +86,27 @@ function importUBCOcTreeMesh(meshfile::AbstractString;Tn::Type{N}=Int64,Tn2::Typ
 
     # create mesh object
     S = sparse3(i1,i2,i3,bsz,(m1,m2,m3))
+    h = [h1,h2,h3]
+    x0 = [x1,x2,x3]
+    return S, h, x0
+end
+
+"""
+    mesh = importUBCOcTreeMesh(meshfile)
+
+    Reads an OcTree mesh in UBC format from disk.
+
+    Input:
+
+        meshfile::AbstractString - File to read
+
+    Output:
+
+        mesh::OcTreeMesh - The mesh
+
+"""
+function importUBCOcTreeMesh(meshfile::AbstractString;Tn::Type{N}=Int64,Tn2::Type{N2}=Int64) where N <: Integer where N2 <: Integer
+    S, h, x0 = importUBCOcTreeMeshLite(meshfile; Tn=Tn, Tn2=Tn2)
     mesh = getOcTreeMeshFV(S,[h1,h2,h3];x0=[x1,x2,x3])
```

Comment:
Tests are currently failing. Looks to be a bug in this line. Should be 
```
mesh = getOcTreeMeshFV(S,h;x0=x0)
```
I believe.

Reply:
My bad.  Should be fixed now.